airbyte-integrations/connectors/source-bamboo-hr/source_bamboo_hr/manifest.yaml

```diff
@@ -0,0 +1,113 @@
+version: 0.65.0
```

Comment:
This is an old version of the CDK. We _require_ that all connectors are made in Builder from scratch when you migrate from low-code. Please rebuild this in builder, running builder locally to use the custom components.